core/src/main/kotlin/org/evomaster/core/output/service/TestSuiteWriter.kt

```diff
@@ -1071,4 +1096,47 @@ class TestSuiteWriter {
             .toList()
     }
 
+    private fun getDtos(): List<DtoClass> {
+        val restSampler = sampler as AbstractRestSampler
+        val result = mutableListOf<DtoClass>()
+        restSampler.getActionDefinitions().forEach { action ->
+            action.getViewOfChildren().forEach { child ->
+                if (child is BodyParam) {
+                    val primaryGene = child.primaryGene()
+                    // TODO: Payloads could also be json arrays, analyze ArrayGene
+                    if (primaryGene is ObjectGene) {
+                        // TODO: Determine strategy for objects that are not defined as a component and do not have a name
```

Comment:
could use something related to `action.getName`, and put this as a comment

Reply:
Could be, yes. We'll most likely come back to this later on